text/0007-gas-price-adjustment.md

```diff
@@ -0,0 +1,134 @@
+# Daily Gas Price Adjustment Algorithm
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0007](https://github.com/casperlabs/ceps/pull/0007)
+
+Gas price is regulated algorithmically. A price floor exists and adjusts daily, based on fullnesses of blocks from the previous day, in order to absorb volatility and reduce congestion.
+
+## Motivation
+
+[motivation]: #motivation
+
+Gas prices are known to be subject to high volatility both in the short term and the long term. This is due to a combination of factors such as inelastic supply, [daily demand cycle](https://solmaz.io/2019/10/21/gas-price-fee-volatility/) and shifting trends. CasperLabs sees these issues with volatility as a key barrier to the adoption of public blockchain infrastructure by businesses. It is also recognized that price stability alone is not enough, and scalability is also a core concern in this arena.
```

Comment:
We should probably find a way to somehow attach external documents to CEPs, or else mostly paste such things into the body itself, because if understanding the CEP requires the external information and your blog goes down, it's no good!

Reply:
There is web archive, but apparently the formulas aren't rendered correctly.
https://web.archive.org/web/20200828124936/https://solmaz.io/2019/02/24/scalable-reward-changing/

Reply:
Link to web archive maybe?
https://web.archive.org/web/20200828125115/https://solmaz.io/2019/10/21/gas-price-fee-volatility/